Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "heroku-ruby-buildpack"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
If you wanted to specify a minimum supported Rust version, you could add `rust-version` here:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field